doc/source/user_guide/io.rst

```diff
@@ -3947,6 +3952,22 @@ in files and will return floats instead.
    Currently pandas only supports *reading* binary Excel files. Writing
    is not implemented.
 
+.. _io.calamine:
+
+Calamine (Excel and ODS files)
+------------------------------
+
+The :func:`~pandas.read_excel` method can read Excel file (``.xlsx``, ``.xlsm``, ``.xls``, ``.xlsb``)
+and OpenDocument spreadsheets (``.ods``) using the ``python-calamine`` module.
+This module is a binding for Rust library `calamine <https://crates.io/crates/calamine>`__
+and faster then other engines in most cases. The semantics and features for reading files
```

Comment:
nit: and is faster than

Is there benchmarks for this?

Reply:
Benchmark from pandas (np.random.randn(2000, 5)):

<details>

```
[75.00%] ··· io.excel.ReadExcel.time_read_excel                                                                                                                                                                                          ok
[75.00%] ··· =============== =============
                  engine                  
             --------------- -------------
                 openpyxl      107±0.4ms  
                   odf          421±2ms   
              calamine_xlsx    23.3±0.1ms 
               calamine_ods   18.3±0.06ms 
             =============== =============

[100.00%] ··· io.excel.ReadExcelNRows.time_read_excel                                                                                                                                                                                     ok
[100.00%] ··· =============== ============
                   engine                 
              --------------- ------------
                  openpyxl     18.8±0.1ms 
                    odf         298±1ms   
               calamine_xlsx   13.8±0.2ms 
                calamine_ods   13.3±0.1ms 
              =============== ===========
```

</details>

Benchmark from pandas (np.random.randn(10000, 50)):

<details>

```
[75.00%] ··· io.excel.ReadExcel.time_read_excel                                                                                                                                                                                          ok
[75.00%] ··· =============== ============
                  engine                 
             --------------- ------------
                 openpyxl     2.87±0.01s 
                   odf        17.1±0.02s 
              calamine_xlsx    551±2ms   
               calamine_ods    742±4ms   
             =============== ============

[100.00%] ··· io.excel.ReadExcelNRows.time_read_excel                                                                                                                                                                                     ok
[100.00%] ··· =============== ============
                   engine                 
              --------------- ------------
                  openpyxl     74.5±0.4ms 
                    odf        12.1±0.03s 
               calamine_xlsx    412±1ms   
                calamine_ods    507±5ms   
              =============== ============
```

</details>

Reading files from https://github.com/MarkPflug/Benchmarks/tree/main/source/Benchmarks/Data (saved as ods in LibreOffice):

<details>

```
[75.00%] ··· io.excel.ReadExcel.time_read_excel                                                                   1/10 failed
[75.00%] ··· ========== ============ ============ ============ ============ =========
             --                                      ext                             
             ---------- -------------------------------------------------------------
               engine       ods          xlsx         xlsm         xlsb        xls   
             ========== ============ ============ ============ ============ =========
              calamine   1.62±0.01s   1.47±0.01s    1.47±0s      926±5ms     956±3ms 
              default      failed      5.31±0s     5.31±0.01s   3.60±0.01s   1.31±0s 
             ========== ============ ============ ============ ============ =========
             For parameters: 'default', 'ods'
             
             
             asv: benchmark timed out (timeout 60.0s)

[100.00%] ··· io.excel.ReadExcelNRows.time_read_excel                                                    1/10 failed
[100.00%] ··· ========== ========= ============= ============= ============= ==========
              --                                      ext                              
              ---------- --------------------------------------------------------------
                engine      ods         xlsx          xlsm          xlsb        xls    
              ========== ========= ============= ============= ============= ==========
               calamine   921±8ms     761±9ms       763±8ms       221±3ms     232±10ms 
               default     failed   13.8±0.09ms   13.8±0.07ms   62.2±0.08ms   794±2ms  
              ========== ========= ============= ============= ============= ==========
              For parameters: 'default', 'ods'
              
              
              asv: benchmark timed out (timeout 60.0s)
```

</details>

openpyxl and pyxlrd support lazy iteration over rows, calamine doesn't support it - https://github.com/tafia/calamine/issues/27.

Reply:
Thanks!